### ISSUE TITLE
Chart.js didn't show labels and data in datasets. 

### ISSUE THREAD

**Issue by oscarxiao** [susanBuck/e28-spring20#82]
Hi Everyone, 
I am wondering if anybody has experience using Chart.js. I used "app.api.find" function to convert into one keys array and values array. #80 . However, when I pass these two arrays into the label and data. The graph won't show anything. 
<img width="707" alt="Screen Shot 2020-04-27 at 9 53 12 PM" src="https://user-images.githubusercontent.com/40640721/80448620-bb4ab800-88d1-11ea-959c-8571092842b3.png">

**Comment by cjreilly** [comment]
I haven't seen the issue but I do know the Google API has an option to export as PNG image. It looks like you still need the Javascript but at least you don't have to worry about positioning chart elements with CSS.

**Comment by ambielecki** [comment]
I think this could be a timing issue.  Your app.api.find method returns a promise which has not resolved by the time `this.fillData()` is called, so `this.expValues` has not been updated.  Try either using async / await, or more simply, move the `this.fillData()` call into the `.then()` callback right after setting `this.expValues`.

**Comment by oscarxiao** [comment]
Thank you for the help Guys!!! Problem solved. The temporary variables I used, *tempExpKeys* and *tempExpValues* are type of array. However, when I assigned these two to *this.expKeys* and *this.expValues*, they became an Object. That is why I can't update the labels and values. 

Then I updated my *fillData function* with two variables and directly pass *tempExpKeys* and *tempExpValues* into the function. 
---
<img width="621" alt="Screen Shot 2020-04-28 at 9 44 35 PM" src="https://user-images.githubusercontent.com/40640721/80561567-c406c100-8999-11ea-9fe5-93cc021929bf.png">

The graph finally shows, but a warning shows. 
---
<img width="773" alt="Screen Shot 2020-04-28 at 9 45 10 PM" src="https://user-images.githubusercontent.com/40640721/80561584-cff28300-8999-11ea-8b09-ebd522f56ff5.png">
